OpenRA.Game/FileSystem/FileSystem.cs

```diff
@@ -243,31 +233,13 @@ public static bool Exists(string name)
 				return MountedFolders.Any(f => f.Exists(name));
 		}
 
-		static Dictionary<string, Assembly> assemblyCache = new Dictionary<string, Assembly>();
-
 		public static Assembly ResolveAssembly(object sender, ResolveEventArgs e)
 		{
 			foreach (var assembly in AppDomain.CurrentDomain.GetAssemblies())
 				if (assembly.FullName == e.Name)
 					return assembly;
 
-			var frags = e.Name.Split(',');
```

Comment:
But I think the safest action for this pr is to just convert this method to use the new FS methods and leave it otherwise unchanged.